rep-2004.rst

```diff
@@ -0,0 +1,836 @@
+REP: 2004
+Title: Package Quality Categories
+Author: William Woodall <william@openrobotics.org>
+Status: Active
```

Comment:
Please vote in reply here with `+1` or `-1` to accept this REP or not, respectively. Thanks!

Reply:
:+1:  This is a well thought-through process and has evolved impressively over the last few months. I think it will lead to maintainers striving for higher quality of their own packages, and maybe it will get some people to actually commit to releasing non-beta versions of their packages :stuck_out_tongue_closed_eyes:

Reply:
:+1: +1

Reply:
I think the copyright policy needs work but other than that 👍

Reply:
+1

Reply:
+1

Reply:
Ok, I think I've addressed the copyright point for now, we can iterate on it further if needed. We have some +1's here and at least no major -1's, so I'm going to merge this (extremely long running) pull request and redirect everything else to new issues.

Thanks everyone!